extensions/blocks/story/player/style.scss

```diff
@@ -0,0 +1,377 @@
+@import '../../../shared/styles/gutenberg-base-styles.scss';
+@import '../../../shared/styles/jetpack-variables.scss';
+
+
+.wp-story-container {
+	height: 500px;
+	margin-left: auto;
+	margin-right: auto;
+	position: relative;
+	list-style: none;
+	padding: 0;
+	z-index: 1;
+	background-color: #0e1112;
```

Comment:
Would be great to move HEXes to SASS variables somewhere for easier maintainability.